glossary/String.md

```diff
@@ -0,0 +1,4 @@
+### String
+
+Strings are one of the primitive data types in JavaScript.
```

Comment:
We should not that it isn't an instance of the class, but will be coerced if needed(which has always been so weird...)